.travis.yml

```diff
@@ -18,13 +19,22 @@ before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 
-  # Install npmv3 because of postinstall bugs in npmv2:
-  # See https://github.com/FormidableLabs/victory/issues/98
-  - npm install -g npm@3
+  - openssl aes-256-cbc -K $encrypted_dbf4ccf442cc_key -iv $encrypted_dbf4ccf442cc_iv
+    -in deploy_static.pem.enc -out deploy_static.pem -d
+
+install:
+  - npm install
 
 script:
   - npm test
+  - NODE_ENV=production node_modules/.bin/builder run build-static
 
-  # Prune deps to just production and ensure we can build the site
-  - npm prune --production
-  - npm run build-static
+after_success:
```

Comment:
Note to self: change this to deploy only on `master`, e.g. https://github.com/FormidableLabs/victory-docs/blob/master/.travis.yml#L34